src/documents/en/host/trouble-shootings.html.md

```diff
@@ -109,3 +109,32 @@ sudo supervisorctl stop cozy-controller
 sudo pkill -9 node
 sudo supervisorctl start cozy-controller
 ```
+
+## I've lost my Cozy password
+
+If you've lost your password, first thing to try is clicking on the `I forgot my password` link to get it sent by email.
+
+If, by any chance, you're hosting cozy on a system with no Email server installed (e.g Postfix), you'll need to access the `CouchDB Relax` page to remove the current user. You now have 2 solutions:
```

Comment:
May be replace `CouchDB Relax` by Futon Web Admin (I take the real name here : http://si-w.co.uk/blog/2013/09/10/couchdb-quickstart/)